styles/main.css

```diff
@@ -1,22 +1,60 @@
 @CHARSET "UTF-8";
-<<<<<<< HEAD
 
-body { background-color: aqua; font-family: 'Domine', serif;}
-h1 { color: grey; background-image: url(../images/gradient.png); background-repeat: repeat-y; font-family: 'Domine', serif;}
-h2 { color: grey; background-image: url(../images/gradient.png); background-repeat: repeat-y; font-family: 'Roboto', sans-serif;}
-a { color: gold; text-decoration:none; }
-ul { color: purple; list-style-type: square;}
-li { color: purple;}
+body { 
+    background-color: aqua; 
+    font-family: 'Domine', serif;
+}

```

Comment:
Notice the indentation here: opening curly brace on its own line, each property on its own line